### PR TITLE
Describing how to add product review from the storefront

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes how to add product review from the storefront

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-my-product-reviews.html

### CODE REVIEW

File: src/customers/account-dashboard-my-product-reviews.md [hunk 1]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+1. Fill in required fields:
+
+   - Fill in the rating stars.
+
+   - Enter the **Nickname**.
+
+   - Describe the **Summary** of the review.
+
+   - Add a detailed description to the **Review** field.
```
Comment (erikmarr) [inline src/customers/account-dashboard-my-product-reviews.md:39]:
```suggestion
   - Detailed description in the **Review** field
```

File: src/customers/account-dashboard-my-product-reviews.md [hunk 2]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+1. Click <span class="btn">Save<span>.
```
Comment (erikmarr) [inline src/customers/account-dashboard-my-product-reviews.md:41]:
```suggestion
1. Clicks <span class="btn">Submit Review<span>.
```

File: src/customers/account-dashboard-my-product-reviews.md [hunk 3]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+{:.bs-callout-info}
+The submitted review will appear on the product page after the Magento admin approves it.
```
Comment (erikmarr) [inline src/customers/account-dashboard-my-product-reviews.md:44]:
```suggestion
The submitted review appears on the product page after the Magento admin approves it.
```

File: src/customers/account-dashboard-my-product-reviews.md [hunk 4]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+- Add **Your Review** for products with existing reviews.
+
+- Be the first to review this product for products with no existing reviews.
+
+1. On the storefront open the **product page**.
+
+1. Scroll down to the details block and select the **Reviews** tab.
+
+1. Fill in required fields:
```
Comment (erikmarr) [inline src/customers/account-dashboard-my-product-reviews.md:31]:
```suggestion
1. Fills in required fields:
```

**Review by erikmarr — CHANGES_REQUESTED** [review]

File: src/customers/account-dashboard-my-product-reviews.md [hunk 5]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
 
     ![]({% link images/images-ee/account-dashboard-my-product-reviews-details.png %}){: .zoom}
     _Review Details_
+
+## Add product review from the storefront
+
+When the **Magento Product Reviews** function is [enabled]({% link icatalog/catalog.md %}), customers can write reviews for any product in your catalog. Reviews can be written from the product page by clicking:
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Hi @erikmarr ☺️
Thank you so much for reviewing.
I have implemented this suggestion, could you please check and approve this?

Thank you!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 6]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+- Add **Your Review** for products with existing reviews.
+
+- Be the first to review this product for products with no existing reviews.
+
+1. On the storefront open the **product page**.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 7]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
 
     ![]({% link images/images-ee/account-dashboard-my-product-reviews-details.png %}){: .zoom}
     _Review Details_
+
+## Add product review from the storefront
+
+When the **Magento Product Reviews** function is [enabled]({% link icatalog/catalog.md %}), customers can write reviews for any product in your catalog. Reviews can be written from the product page by clicking:
+
+- Add **Your Review** for products with existing reviews.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 8]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+- Be the first to review this product for products with no existing reviews.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 9]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+1. On the storefront open the **product page**.
+
+1. Scroll down to the details block and select the **Reviews** tab.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 10]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+1. Fill in required fields:
+
+   - Fill in the rating stars.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 11]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+   - Enter the **Nickname**.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md [hunk 12]
```diff
@@ -15,3 +15,30 @@ _My Product Reviews_
...
+
+   - Describe the **Summary** of the review.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

File: src/customers/account-dashboard-my-product-reviews.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-my-product-reviews.md]: Done, please check!

Thanks!

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@erikmarr the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1217`.

**Pull request merged**